portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -46,8 +59,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String commentText = request.getParameter("comment-input");
     long timestamp = System.currentTimeMillis();
 
-    comments.add(commentText);
-
+    // comments.add(commentText);
```

Comment:
No need for this line because you are writing to the datastore now. You also redirect users to the index page which will reload the page, call the onload method, make a GET request, which will then read the datastore and show it in the page.